webdriver-extension/index.bs

```diff
@@ -453,8 +452,8 @@ The <a>remote end steps</a> are:
   </tbody>
 </table>
 
-The <dfn>get mock sensor</dfn> <a>extension command</a> simulates user
-getting information of a given type of [=mock sensor=].
+The <dfn>get mock sensor</dfn> <a>extension command</a> simulates
```

Comment:
Quote [conversations ](https://github.com/w3c/sensors/pull/369/files/279d6e0b0eaff3906524ab525994421f91d335a0#r234730684)with @JohnChen0:
>John: The word "simulates" seems wrong here. This extension command creates a mock sensor, not just simulate creating one.
Wanming: This actually "simulates" user action to create a mock sensor, and there are two instances that use "simulates" in Permission API and Reporting API. Does that make sense to you?
John: Sounds good.

So, if we want to erase user action, we'd better avoid using "simulates".

Reply:
I've pushed a new version that removes "simulates" from most places -- I think John's point is that we are really manipulating a (mock) sensor rather than pretend to do that. Let me know if it looks OK now.